package_picks/package-pick-8.19~2024.01+beta1.sh

```diff
@@ -49,7 +49,7 @@ PACKAGES=""
 # Coq needs a patched ocamlfind to be relocatable by installers
 PACKAGES="${PACKAGES} PIN.ocamlfind.1.9.5~relocatable"  # TODO port patch to 1.9.6
 # Since dune does support Coq, it is explicitly selected
-PACKAGES="${PACKAGES} dune.3.12.1"
+PACKAGES="${PACKAGES} dune.3.15.2"
```

Comment:
Didn't we say we use 3.15.2 on Windows only and 3.13.X elsewhere, because 3.14/3.15 has severe bugs? Or are these fixed in 3.15.2?

Reply:
The fix was pushed in 3.15.2 (which was merged today)

Reply:
See here: https://github.com/ocaml/opam-repository/pull/25714#event-12653673866